src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -141,10 +141,13 @@ export class CalciteTipManager {
 
   setUpTips(): void {
     const tips = Array.from(this.el.querySelectorAll("calcite-tip"));
+    this.total = tips.length;
```

Comment:
Just an observation, but is there a difference between checking `this.total` and `this.tips.length`?

Reply:
functionally no. Performance-wise, it prevents a second lookup to length. Not winning any awards with that optimization, but its a small win.